assets/js/blocks/product-categories/block.js

```diff
@@ -142,6 +152,24 @@ const ProductCategoriesBlock = ( { attributes, setAttributes, name } ) => {
 							setAttributes( { hasEmpty: ! hasEmpty } )
 						}
 					/>
+					{ isSiteEditor && (
+						<ToggleControl
+							label={ __(
+								'Only show children of current category',
+								'woo-gutenberg-products-block'
+							) }
+							help={ __(
+								'This will effect product category pages',
```

Comment:
I think there is a typo here:

```suggestion
								'This will affect product category pages',
```